sys/socket.h

```diff
@@ -4,10 +4,14 @@ typedef int socklen_t;
 #define MSG_DONTWAIT    0x40 
 #define AF_LOCAL        1       /* POSIX name for AF_UNIX       */
 #define AF_INET         2       /* Internet IP Protocol         */
+
+#ifndef WIN_IOVEC
+#define WIN_IOVEC
```

Comment:
You can see the same guard here: https://github.com/codilime/contrail-vrouter/pull/49/files#diff-c53c11db10feba04a70725fc929f8637